### DISCUSSION TITLE
bottomSheet- $q.bottomSheet is not a function error

### DISCUSSION THREAD

**Discussion by Eyuel-Nigussie** [quasarframework/quasar#15461] [General - Components / Directives / etc] [answered]
I tried to use a Bottomsheet. I copied the code from the docs. But on the console it is showing error "runtime-core.esm-bundler.js:220 Uncaught TypeError: $q.bottomSheet is not a function ".

**Comment by metalsadman** [comment, marked as answer]
You need to install it https://quasar.dev/quasar-plugins/bottom-sheet#installation

**Reply by Eyuel-Nigussie** [reply]
So silly of me not to see it,I must have brushed it off. Thanks
